spec/bin/query-cloudwatch_spec.rb

```diff
@@ -12,6 +12,7 @@
 
     before do
       allow(QueryCloudwatch).to receive(:exit)
+      allow(Reporting::CloudwatchClient).to receive(:MAX_RESULTS_LIMIT).and_return(10_000)
```

Comment:
I'd use `stub_const` here instead of stubbing the method

Reply:
> I'd use `stub_const` here instead of stubbing the method

👍 Updated in 0ac257c93.